cmd/minikube/cmd/delete.go

```diff
@@ -87,6 +87,24 @@ func (error DeletionError) Error() string {
 	return error.Err.Error()
 }
 
+var DeleteHostAndDirectoriesGetter = func(api libmachine.API, cc *config.ClusterConfig, profileName string) error {
```

Comment:
this func is not used anywhere in the code, lets make it private, also name Getter doesnt sound right here.

how about
var hostAndDirsDeleter = func ( ...